templates/components/argument.js.sapphire

```diff
@@ -1,15 +1,23 @@
 { "category": "arguments" }
 ---
-import { Argument } from "@sapphire/framework";
+const { Argument } = require('@sapphire/framework');
 
-export class UserArgument extends Argument {
-    constructor(context, options) {
-        super(context, {
-            ...options
-        });
-    }
+class UserArgument extends Argument {
+	/**
+	 * @param {Argument.Context} context
+	 * @param {Argument.Options} options
+	 */
+	constructor(context, options) {
+		super(context, {
+			...options
+		});
```

Comment:
```suggestion
	 */
	constructor(context) {
		super(context, {
			// any options you want here
		});
```

Reply:
It is better to make people spread options in case they create their own class extensions and end up quashing their own options. We also use spread syntax in the guide.

Reply:
Thats not this. This is the user extending *our* Command class which lacks any options being passed down.

Reply:
Always assume the lowest common denominator. If someone makes `MyBotCommnad` with some default options and copying one of their own existing and generated commands then it's better if they copy one with spread than one without.

And again the argument of sticking to what we write on the guide.

Reply:
I really don't see this argument standing lol. If we want this we should be making a guide for it, not making our templates do useless spreads